feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/session_proxy_relay.go

```diff
@@ -111,18 +135,17 @@ func (s *RelayRouter) TerminateSession(
 	ctx context.Context, r *protos.SessionTerminateRequest) (*protos.SessionTerminateResponse, error) {
 
 	// TerminateSession's SID is just IMSI with "IMSI" prefix which findServingNHFeg() should remove
-	client, ctx, cancel, err := s.getSessionProxyClient(ctx, r.GetCommonContext().GetSid().GetId())
+	client, ctx, cancel, err := s.getProxyClient(ctx, r.CommonContext.RatType, r.GetCommonContext().GetSid().GetId())
 	if err != nil {
 		return nil, err
 	}
 	defer cancel()
 	return client.TerminateSession(ctx, r)
 }
 
-func (s *RelayRouter) getSessionProxyClient(
-	c context.Context, imsi string) (protos.CentralSessionControllerClient, context.Context, context.CancelFunc, error) {
-
-	conn, ctx, cancel, err := s.GetFegServiceConnection(c, imsi, FegSessionProxy)
+// Get Proxy Client based on RatType
+func (s *RelayRouter) getProxyClient(c context.Context, r protos.RATType, imsi string) (protos.CentralSessionControllerClient, context.Context, context.CancelFunc, error) {
```

Comment:
Please use more specific naming for RelayRouter methods, relay router is shared between multiple FeG Proxies, they are all proxies - swx, s6a, session_proxy... getProxyClient may imply any of them and potentially conflict with any of them

Reply:
Done.